azure-pipelines-integration-corehost.yml

```diff
@@ -85,7 +85,6 @@ stages:
     configuration: Debug
```

Comment:
I think we can delete this now, as well as the [corresponding pipeline](https://dev.azure.com/dnceng-public/public/_build?definitionId=100&_a=summary). Need to check with @dotnet/roslyn-infrastructure on how to do it properly